superset/connectors/sqla/models.py

```diff
@@ -229,6 +230,30 @@ class TableColumn(Model, BaseColumn, CertificationMixin):
     update_from_object_fields = [s for s in export_fields if s not in ("table_id",)]
     export_parent = "table"
 
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
See https://docs.sqlalchemy.org/en/13/orm/constructors.html. Note I'm not overly sure if `super` is necessary—it's not included in the example, but I gather it doesn't hurt.